net.c

```diff
@@ -63,7 +63,15 @@ int redisNetRead(redisContext *c, char *buf, size_t bufcap) {
         if ((errno == EWOULDBLOCK && !(c->flags & REDIS_BLOCK)) || (errno == EINTR)) {
             /* Try again later */
             return 0;
-        } else {
+        }
+		#ifdef _WIN32
+		else if(errno == ETIMEDOUT && (c->flags & REDIS_BLOCK))
+		{	/* especially in windows */
```

Comment:
Can you fix the formatting here? (spaces, no tabs; opening brace on same line, etc.)

Reply:
I also don't see how this block is specific to Windows.

Reply:
> I also don't see how this block is specific to Windows.

In fact, I think so. I added the block in order to clarify a timeout error code when giving the timeout value on blocking mode. But, originally the block wasn't there and I was uncertain that I coded correct, so I specified the block to only Windows.

If this block doesn't have any problem, I can remove #preprocessor or else I will remove the block.